portfolio/src/main/webapp/script.js

```diff
@@ -21,8 +21,17 @@ function addRandomGreeting() {
 
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+async function loadingcontent() {
+    const responseFromServer = await fetch('/hello');
+    const textdisplayed = await responseFromServer.text();
```

Comment:
similar comment: See https://google.github.io/styleguide/jsguide.html#naming-method-names, method names should be written in lowerCamelCase like "lowerCamelCase"

Reply:
Done